### PR TITLE
Reformat & revisions for pre-Kona mailing

### PR DESCRIPTION
**Pull request by mhoemmen**
More revisions will follow, but this is a start.

### CODE REVIEW

**Comment by jayeshbadwaik** [comment]
@mbrucher  The text you [committed](https://github.com/cpp-linear-algebra/prior-art/commit/06f9d410cf87b57c7758e4eebc0bbccda4d88de9) has been modified by @mhoemmen  in a commit [here](https://github.com/cpp-linear-algebra/prior-art/pull/5/commits/7e0408eed570dfcc28bf5805a1721255eab244a1). As per the policy, it would be nice to have a quick sanity check from you about the content. Thank you. :-)

**Comment by mhoemmen** [comment]
@mbrucher I might want to expand your text a bit more with some lessons learned -- if you have any thoughts, please feel free to post them here :-)

File: brief-history.tex [hunk 1]
```diff
@@ -931,77 +931,124 @@ \subsection{GraphBLAS}
...
 \label{S:Python}
 
-Scientists saw Python in the 2000s as an alternative for \texttt{Matlab}, itself
- an alternative interpreted langage to Fortran. Despite not having native
-  support for multidimensional arrays, two main projects, \texttt{Numarray} and
-  \texttt{Numeric}, were merged by Travis Oliphant to create
-  \texttt{Numpy}~\cite{oliphant2006numpy}. The communities behind
-  the two other projects soon merged.
+Summary: The Python programming language does not come with native
+linear algebra or multidimensional array support, but Numpy offers a
```
Comment (mbrucher) [inline brief-history.tex:979]: Actually, it does (see https://docs.python.org/3/c-api/buffer.html). Numpy builds now on top of it.

**Review by mbrucher — COMMENTED** [review]
I will think about this this weekend. I just have one note, otherwise very happy with the changes.

File: brief-history.tex (hunk 1 above)
Comment (mhoemmen) [reply brief-history.tex:979]: @mbrucher Should we elaborate this point, or just say something like the following: "Python did not come with native multidimensional array support until version 3 of the language, but Numpy..."?

**Comment by mhoemmen** [comment]
@mbrucher wrote:
> I will think about this this weekend. I just have one note, otherwise very happy with the changes.

If I don't hear from you by Sunday, I will need to submit the paper, since it is due then.  I will continue development on my fork.

**Comment by jayeshbadwaik** [comment]
I will merge these changes for now, the rest can be worked out in another pull request. It helps keeps the document merging easy.

**Comment by mhoemmen** [comment]
@jayeshbadwaik Thanks for the merge!  I have some more contributions coming in; I didn't write them, but the authors want me to add the changes.  I'll add those today or tomorrow at the latest.

File: brief-history.tex (hunk 1 above)
Comment (mhoemmen) [reply brief-history.tex:979]: @mbrucher Buffer is a protocol for Python's C API.  Python 3 doesn't have multidimensional arrays in the Python language, as far as I can tell.  Is that right?

Comment (mbrucher) [reply brief-history.tex:979]: I guess it is. It's a little bit fuzzy I have to say, as this C-API compatibility is central now for lots of array types (numpy, pillow, TF...)

Comment (mhoemmen) [reply brief-history.tex:979]: @mbrucher Thanks for the comment! :-) While I use Python sometimes, I'm not an expert.  Would you like to write this bit, or should I just try?  I could add what you just said, that many Python libraries use the Buffer protocol to implement array types.

Comment (mbrucher) [reply brief-history.tex:979]: I'll try to write something tomorrow, I have to take some time on SG19 paper, I'll do both at the same time. Thanks a lot for everything you already did!

**Pull request merged**